src/components/dashboardBuilder/index.jsx

```diff
@@ -5,23 +5,23 @@ import Dashboard from './Dashboard'
 FSBL.addEventListener("onReady", function () {
 	ReactDOM.render(<Dashboard />,
 		document.getElementById('app'), () => {
-			async function spawnCanvas() {
-				var canvasAttributes = {
-					position: "unclaimed",
-					left: "25",
-					right: "25",
-					bottom: "25",
-					height: "400",
-					width: "400",
-					dockOnSpawn: false
-				};
+			// async function spawnCanvas() {
+			// 	var canvasAttributes = {
+			// 		position: "unclaimed",
+			// 		left: "25",
+			// 		right: "25",
+			// 		bottom: "25",
+			// 		height: "400",
+			// 		width: "400",
+			// 		dockOnSpawn: false
+			// 	};
 
-				await FSBL.Clients.LauncherClient.spawn("dashboardCanvas", canvasAttributes, (err, response) => { });
-			}
+			// 	await FSBL.Clients.LauncherClient.spawn("dashboardCanvas", canvasAttributes, (err, response) => { });
+			// }
 
-			FSBL.addEventListener("onReady", function () {
-				spawnCanvas();
+			FSBL.addEventListener("onReady", () => {
```

Comment:
It looks like this onReady listener inside of an onReady listener now doesn't do anything, is it still needed?